util/log/clog.go

```diff
@@ -739,22 +739,30 @@ func (l *loggingT) processForStderr(s severity, data []byte) []byte {
 	default:
 		return data
 	}
-	idx := bytes.Index(data, []byte("]"))
-	if idx == -1 {
+	if len(data) < 5 {
 		return data
 	}
 	var prefix []byte
 	switch s {
+	case infoLog:
+		prefix = []byte("\033[38;5;33m") // blue
 	case warningLog:
-		prefix = []byte("\033[0;31;49m") // red
+		prefix = []byte("\033[38;5;214m") // orange
```

Comment:
Hmm. OK, will need to flesh out our handling of this a little. None of the
existing term coloring packages seem to handle the actual term type.

On Wed, May 20, 2015 at 5:06 PM, Ben Darnell <notifications@github.com>
wrote:

> In util/log/clog.go
> <https://github.com/cockroachdb/cockroach/pull/1083#discussion_r30749351>:
>
> >  	case warningLog:
> > -		prefix = []byte("\033[0;31;49m") // red
> > +		prefix = []byte("\033[38;5;214m") // orange
>
> I think these escapes are unique to the xterm-256color terminal type, so
> we shouldn't use them on terminals that just advertise "xterm" or "ansi".
> We should either limit ourselves to the old-school 16-color palette or
> tighten our $TERM checking to only match xterm-256color.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/1083/files#r30749351>.
>